openapi/schemas/processes-job-management/inputs.yaml

```diff
@@ -0,0 +1,21 @@
+type: object
+required:
+  - inputs
+  - outputs
+properties:
+  inputs:
+    type: object
+    additionalProperties:
+      $ref: "../processes-workflow/input-workflow.yaml"
+  outputs:
+    type: object
+    additionalProperties:
+      $ref: "../processes-workflow/output-workflow.yaml"
```

Comment:
@gfenoy 
While testing/implementing more of the OAP v1 vs v2 execution modes, I found out that the following should probably be defined as well: 

- V1:
  - `response=raw|document`
  - `outputs` -> must be extended to include `transmissionMode=value|reference`
  - `headers`
    - `Accept`
- V2: 
  - `headers`, notably for:
    - `Prefer: return=minimal|representation`
    - `Prefer: respond-async|wait=x` 
    - `Accept`

Basically, the `inputs` and `outputs` themselves are not always enough to replicate a job execution because of the various execution parameters and content negotiation that can drastically affect how the results are returned (encodings, media-type, value/link, etc.).